src/jit/codegencommon.cpp

```diff
@@ -11276,6 +11276,25 @@ bool Compiler::IsRegisterPassable(GenTreePtr tree)
 {
     return IsRegisterPassable(gtGetStructHandleIfPresent(tree));
 }
+
+// Returns true if the type is returned in multiple registers
+// 
+// Arguments
+//     hClass   -  type handle
+//
+// Returns
+//     true if type is returned in multiple registers
+bool Compiler::IsMultiRegRetVal(CORINFO_CLASS_HANDLE hClass)
+{
```

Comment:
Seems weird to have the word "Val" indicate a type. Maybe instead IsTypeMultiRegRet?